source/_patterns/molecules/interactive/accordion/accordion.twig

```diff
@@ -0,0 +1,24 @@
+{% set tag = tag|default('h2') %}
+{% set classes = [
+  'usa-accordion',
+  'usa-accordion--bordered'
+  ]|merge(classes|default([]))
+%}
+
+<div{{ attributes.addClass(classes) }}>
+  {% for item in accordion %}
+    {% set expanded = loop.first ? 'true' : 'false' %}
+    {% set id = 'accordion-item-' ~ loop.index ~ '-' ~  random() %}
+
+    {# Use `tag` to pass the accurate heading level to maintain the document outline #}
+    <{{ tag }}{{ item.title_attributes.addClass('usa-accordion__heading')}}>
```

Comment:
You'll notice `title_attributes` and `content_attributes` are set in the data at item level. The ones we set globally in `data.json` would not apply here, so they need to be created. Another way of doing this, which has the same result, is the original way you had it:

```
{% set accordion_heading_attributes = accordion_heading_attributes ? accordion_heading_attributes : create_attribute() %}
{% set accordion_content_attributes = accordion_content_attributes ? accordion_content_attributes : create_attribute() %}
``` 
However, I suspect you might like the data approach more, since it looks a lot cleaner.  Both ways work.

P.S. - I'm not sure we really the attributes at this level, but I wanted to show you how to do this anyway, and we'll see if we can drop these later when we get further along.